### PR TITLE
feat: Generate service mutations dynamically

### PR DESCRIPTION
**Pull request by mfranceschit**
## Issue tracker links

[EP-1631](https://autoclouddev.atlassian.net/browse/EP-1631)

## Changes/solution

- Generates mutations (upsert/delete) dynamically for services.

It will be require for smart inserts.

### CODE REVIEW

File: src/services/index.ts [hunk 1]
```diff
@@ -824,14 +826,14 @@ export default class Provider extends CloudGraph.Client {
           result.entities[existingServiceIdx] = {
             className: serviceClass.constructor.name,
             name: serviceData.name,
-            mutation: serviceClass.mutation,
+            mutation: generateEntityMutations(schemaName),
```
Comment (tyler-dunkel) [inline src/services/index.ts:829]: So doe this change `mutation` to be an object `{ add: ...., delete: ... }`?

Comment (tyler-dunkel) [inline src/services/index.ts:829]: So doe this change `mutation` to be an object `{ add: ...., delete: ... }`?

Comment (tyler-dunkel) [reply src/services/index.ts:829]: And if so, can we delete the mutation files?

Comment (tyler-dunkel) [reply src/services/index.ts:829]: And if so, can we delete the mutation files?

Comment (mfranceschit) [reply src/services/index.ts:829]: Yes, that's the case. Those files won't be required anymore. I didn't know if include that as part of this PR, but I can delete them if you want.

Comment (tyler-dunkel) [reply src/services/index.ts:829]: Lets make sure this strategy works first so we can easily rollback if not but ok sounds good!

**Pull request closed without merging**